src/components/bulk-email-tool/BulkEmailTool.jsx

```diff
@@ -18,14 +21,24 @@ export default function BulkEmailTool() {
         <div>
           <NavigationTabs courseId={courseId} tabData={courseMetadata.tabs} />
           <BulkEmailProvider>
-            <div>
+            <Container size="md">
```

Comment:
is this from the mock? a size of `md` here makes the container a bit small on larger screens: 
![image](https://user-images.githubusercontent.com/16495365/176022899-ac53bca5-ff21-4bb7-9c8f-da4c40642aeb.png)


if we can bump up the size to `xl` its more in line with the nav bar sizing:
![image](https://user-images.githubusercontent.com/16495365/176023164-a6fb5228-350c-4f29-aace-4815c8b26280.png)

(I can also ask UX if you're unsure)

Reply:
Yes he was saying he liked the medium width. I agree that on larger screens it looks a little better. I think the reason was that we don't NEED all that room for the email text and there may also be some UX thing about the width of text and how easy it is to read(?)

Reply:
Works for me! 👍